src/Features/LanguageServer/ProtocolUnitTests/Configuration/DidChangeConfigurationNotificationHandlerTest.cs

```diff
@@ -104,6 +105,48 @@ public class A { }";
             VerifyValuesInServer(server.TestWorkspace, clientCallbackTarget.MockClientSideValues);
         }
 
+        [Fact]
+        public void VerifyLspClientOptionNames()
+        {
+            var actualNames = DidChangeConfigurationNotificationHandler.SupportedOptions.Select(
+                DidChangeConfigurationNotificationHandler.GenerateFullNameForOption).ToArray();
+            // These options are persist in the LSP client. Please make sure also modify the LSP client code if these strings are changed.
+            var expectedNames = new[]
+            {
+                "symbol_search.dotnet_search_reference_assemblies",
```

Comment:
Should we have this sorted and use one of our sort-ignoring Assert functions?

Reply:
Yeah it's possible